packages/foreman/foreman/foreman.spec

```
@@ -737,6 +739,10 @@ webpack --bail --config config/webpack.config.js
 rm config/database.yml config/settings.yaml
 
 %install
+%if 0%{?fedora} || 0%{?rhel} >= 8
+grep -lr '#!/usr/bin/python' extras/noVNC | xargs sed -i 's/#!\/usr\/bin\/python/#!\/usr\/bin\/python3/g'
```

Why is the grep needed? Can't you just sed it? I'd also use pipes as separators so you don't need to escape everything.
```suggestion
sed -i 's|#!/usr/bin/python|#!/usr/bin/python3|g' extras/noVNC
```

On a side note: it'd be great if we could actually depend on websockify as a standalone package and solve it there. That would also update our bundled copy by a lot.

---

I think there is also the use of `/usr/bin/env python` in a few of the files. So need to account for each.

---

I now see that extras/noVNC is a directory, not a file. Still, the comment about the pipes stands.

---

@ekohl Thanks for the advice!